srcpkgs/grass/files/grass.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Grass
+Comment=
+Exec=grass78 -wxpython  %f
+Icon=/usr/lib/grass78/gui/icons/grass-48x48.png
+Terminal=true
+Type=Application
+Categories=Science;Education;
```

Comment:
It's an open source project, please ask them to provide one (you can even PR this their way) instead of vendoring it in.

Reply:
???

Reply:
vergiss es